src/main/java/alexiil/mc/lib/multipart/impl/PartContainer.java

```diff
@@ -1103,7 +1104,7 @@ void onListenerRemoved(SingleListener<?> single) {
         }
 
         if (property instanceof RedstonePowerProperty) {
-            getMultipartWorld().updateNeighbors(getMultipartPos(), LibMultiPart.BLOCK);
+            updateOwnNeighbours();
```

Comment:
Okay maybe not here...
```suggestion
            getMultipartWorld().updateNeighbors(getMultipartPos(), LibMultiPart.BLOCK);
```